doc/modules/ROOT/pages/tutorials/fastrp-and-knn.adoc

```diff
@@ -2,6 +2,10 @@
 
 = Product recommendations with kNN based on FastRP embeddings
 
+This Jupyter notebook is hosted
+https://github.com/neo4j/graph-data-science-client/blob/main/examples/fastrp-and-knn.ipynb[here]
+in the Neo4j Graph Data Science Client Github repository.
+
```

Comment:
Did you write this manually, or got this part through the generator?

Reply:
This goes through the generator